docs/discover/get-started-discover.asciidoc

```diff
@@ -0,0 +1,354 @@
+
+. Find **Discover** in the navigation menu or by using the <<kibana-navigation-search,global search field>>.
+
+. Select the data view that contains the data you want to explore.
++
+TIP: {kib} requires a <<data-views,{data-source}>> to access your Elasticsearch data. A {data-source} can point to one or more indices, {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases]. When adding data to {es} using one of the many integrations available, some data views are created automatically most of the time, but you can also create your own.
++
+If you're using sample data, data views are automatically created and are ready to use.
++
+[role="screenshot"]
+image::images/discover-data-view.png[How to set the {data-source} in Discover, width="40%"]
+
+. If needed, adjust the <<set-time-filter,time range>>, for example by setting it to the *Last 7 days*.
++
+The range selection is based on the default time field in your data.
+If you are using the sample data, this value was set when you added the data.
+If you are using your own data, and it does not have a time field, the range selection is not available.
+
+**Discover** is populated with your data and you can view various areas with different information:
+
+* All fields detected are listed in a dedicated panel.
+* A chart allows you to visualize your data. 
+* A table that shows all the data matching your data view.
+By default, the table includes a column for the time field and a column that lists all other fields in the document.
+You can modify the document table to display your fields of interest.
+
+You can later filter the data that shows in the chart and in the table by specifying a query and changing the time range.
+
+[float]
+[[explore-fields-in-your-data]]
+=== Explore the fields in your data
+
+**Discover** provides utilities designed to help you make sense of your data:
+
+. In the sidebar, check the available fields. It's very common to have hundreds of fields. Use the search at the top of that sidebar to look for specific terms in the field names.
++
+In this example, we've entered `ma` in the search field to find the `manufacturer` field.
++
+[role="screenshot"]
+image:images/discover-sidebar-available-fields.png[Fields list that displays the top five search results, width=40%]
++
+TIP: You can use wildcards in field searches.  For example, `goe*dest` finds `geo.dest` and `geo.src.dest`.
+
+. Select a field to view its most frequent values.
++
+**Discover** shows the top 10 values and the number of records used to calculate those values.
+
+. Select the *Plus* icon to add fields to the results table.
+You can also drag them from the list into the table.
++
+[role="screenshot"]
+image::images/discover-add-icon.png[How to add a field as a column in the table, width="50%"]
++
+When you add fields to the table, the **Summary** column that was summarizing each result's data is replaced.
```

Comment:
```suggestion
When you add fields to the table, the **Summary** column is replaced.
```
I'd actually keep it simple to avoid making this sentence confusing or too complicated